nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/resources/docs/org.apache.nifi.processors.standard.ExecuteStreamCommand/additionalDetails.html

```diff
@@ -0,0 +1,166 @@
+    </head>
+
+    <body>
+        <h2>Description</h2>
+        <p>The ExecuteStreamCommand processor provides a flexible way to integrate external commands and scripts into NiFi data flows.
+        ExecuteStreamCommand can pass the incoming FlowFile's content to the command that it executes similarly how piping works.</p>
+
+        <h2>Configuration options</h2>
+
+        <h3>Working Directory</h3>
+        <p>If not specified, NiFi root will be the default working directory.</p>
+
+        <h3>Configuring command arguments</h3>
+        <p>The ExecuteStreamCommand processor provides two ways to specify command arguments: using Dynamic Properties and the Command Arguments field.</p>
+
+        <h4>Command Arguments field</h4>
+        <p>This is the default. If there are multiple arguments, they need to be separated by a character specified in the Argument Delimiter field.
+        When needed, '-' and '--' can be provided, but in these cases Argument Delimiter should be a different character.</p>
+
+        <p>Consider that we want to list all files in a directory which is different from the working directory:</p>
+
+        <table>
+            <tr>
+                <th>Command Path</th>
+                <th>Command Arguments</th>
+            </tr>
+            <tr>
+                <td>ls</td>
+                <td>-lah;/path/to/dir</td>
+            </tr>
+        </table>
+
+        <p><b>NOTE:</b> the command should be on <code>$PATH</code> or it should be in the working directory, otherwise path also should be specified.</p>
+
+        <h4>Dynamic Properties</h4>
+        <p>Arguments can be specified with Dynamic Properties. Dynamic Properties with the pattern of 'command.arguments.&lt;commandIndex&gt;' will be appended
+        to the command in ascending order.</p>
+
+        <p>The above example with dynamic properties would look like this:</p>
+
+        <table>
+            <tr>
+                <th>Property Name</th>
+                <th>Property Value</th>
+            </tr>
+            <tr>
+                <td>command.arguments.0</td>
+                <td>-lah</td>
+            </tr>
+            <tr>
+                <td>command.arguments.1</td>
+                <td>/path/to/dir</td>
+            </tr>
+        </table>
+
+        <h3>Configuring environment variables</h3>
+        <p>In addition to specifying command arguments using the Command Argument field or Dynamic Properties, users can also use environment variables with
+        the ExecuteStreamCommand processor. Environment variables are a set of key-value pairs that can be accessed by processes running on the system.
+        ExecuteStreamCommand will treat every Dynamic Property as an environment variable that doesn't match the pattern 'command.arguments.&lt;commandIndex&gt;'.</p>
+
+        <p>Consider that we want to execute a maven command with the processor. If there are multiple Java versions installed on the system, you can specify
```

Comment:
`Maven` should be capitalized.
```suggestion
        <p>Consider that we want to execute a Maven command with the processor. If there are multiple Java versions installed on the system, you can specify
```